### PR TITLE
Set FC map as a multi flatmap so it has dynamic markers

### PR DESCRIPTION
**Pull request by Tehsurfer**
This was done by moving the functions needed for marker updates into a mixin and then using them on the flatmap

image

### CODE REVIEW

File: src/mixins/ContentMixin.js [hunk 1]
```diff
@@ -310,6 +310,12 @@ export default {
     requestSynchronisedEvent: function () {
       return;
     },
+    flatmapReadyCall: function (flatmap) {
```
Comment (alan-wu) [inline src/mixins/ContentMixin.js:313]: Can this be moved to the Flatmap.vue? I tried not to expose method here unless it is used by more than one module.

**Review by alan-wu — CHANGES_REQUESTED** [review]
It looks good, can you remove markerZoomLevels.js or markerZoomLevelsHardCoded.js please? They are identical but used in separated places.

File: src/mixins/ContentMixin.js (hunk 1 above)
Comment (Tehsurfer) [reply src/mixins/ContentMixin.js:313]: yeah can do :)

**Comment by alan-wu** [comment]
Thanks for removing markerZoomLevels but ContentMixin imports it and the new changes do not include renaming the file to markerZoomLevelsHardCoded. Please see line 8 in ContentMixin. Thanks.

**Review by alan-wu — APPROVED** [review]
Looks good to me.

**Pull request merged**